### ISSUE TITLE
Manual "expanded" state management

### ISSUE THREAD

**Issue by alexandermorgan** [springload/react-accessible-accordion#304]
Thank you for this tool! It has facilitated making an accessible form for me.
In your readme file you state that if we have a valid use case for "items which require manual 'expanded' state-management", then we are welcome to comment. I think this would be a very useful addition to your accordion and would mostly be used to the benefit of your stated design goal to "require bespoke interactive mechanisms in order to expand, collapse and 'disable'" accordion items. While programmatic expansion and collapsing of accordion items would make it possible to create an accordion that does not respect best practices in accessibility, I doubt that many people would use your library in this way.
In my use case, I have an accordion with 5 panels limited to one open at a time. Each panel contains questions for the user to fill out about the environmental policies of a cafe. If the user wants to reset all of their answers, I offer this option via a "Reset" button below the accordion. This reset button works, but if an accordion panel was open, it will remain open on reset which is a little counter-intuitive for the user. Since I am only trying to close all the panels when the user pushes a button, I would still be respecting your goal of requiring a bespoke interactive mechanism to collapse the panels. I think this is the way most developers would use programmatic setting of the expanded state since they could use a different accordion if respecting accessibility best practices wasn't important to them.
As a work-around, I'm currently putting the accordion inside a div whose key argument I change to force the accordion to reinitialize.

**Comment by holloway** [comment]
Hi @alexandermorgan  .. the docs were outdated and we've since added support for that feature

https://github.com/springload/react-accessible-accordion#dangerouslysetexpanded-boolean-optional

It does however sound that the `key` approach that you're using is better as it forces a reset without having to consider manual state management.

**Comment by alexandermorgan** [comment]
I did see this dangerouslySetExpanded setting but it leaves the accordion in a static state so the user can no longer expand or collapse any of the panels. This effectively turns the accordion into an image and this may be useful in certain cases, but it does not address my use case as described above. FYI, I also created a [stackoverflow question](https://stackoverflow.com/questions/65082310/programmatically-collapse-all-react-accessible-accordion-panels/65117314#65117314) about this issue.
